Bluedot-Innovation/OracleMinimalApp-Android
# Po/bd 2359 simple init

To test:
1. Login with Oracle Responsys account
2. Navigate to the **App Develop Console** and select there your android project. Then download **pushin_config.json** under the **Download buuton** and paste into `app/src/main/res/` folder.
3. Download **google-services.json** from *OracleResponsysDemo* Firebase project
4. Update Project ID in `MainActivity.java`
5. trigger a geofence
6. within a few minutes observe a push notification and [Programs count updated](https://bluedotinnovation.atlassian.net/wiki/spaces/ENG/pages/1466925069/How+to+test+Oracle+Minimal+App+Integration)
